aequilibrae/project/data/matrices.py

```diff
@@ -77,7 +77,7 @@ def update_database(self) -> None:
                 while f'{name}_{i}' in self.__items:
                     i += 1
                 name = f'{name}_{i}'
```

Comment:
is it saving matrices with increasing number, right? aaa_0, aaa_1. If so, it perhaps helps the user the zfill on the numbers so that it stays sorted. If this doesn't make sense, please ignore it :-)

Reply:
There is no need for matrices to be in order.  This  is just a way to generate an arbitrary matrix name when all the user does is throw the matrix in the appropriate folder